### ISSUE TITLE
Where I've been and article

### ISSUE THREAD

**Issue by chocmake** [chocmake/chocmake.github.io#1]
Hey, been away from Discord due to verification rejection over a false positive. Decided to make an [article](https://chocmake.github.io/articles/discord-false-positives/) about it since for users exclusive to Discord it makes it tricky to get in touch.

 [![](https://raw.githubusercontent.com/chocmake/chocmake.github.io/refs/heads/main/articles/discord-false-positives/meta/embed.png)](https://chocmake.github.io/articles/discord-false-positives/)

> Best viewed in Chromium/Firefox as WebKit/Safari has truly awful, long standing SVG compatibility issues so most images have been made into static, non-animated rasters for the latter (relying on non-local testing so apologies if anything more is off-kilter on Webkit).

Sorry for the pings, just thought I'd notify some users I could think of that share active Github accounts from a few communities I'm part of to share the update, hope no one minds.

@kapuragu [MGSV], @originalnicodr / @Fraulk [Framed], @SilverEzredes [Cyberpunk], @notNSANE [Vivaldi]

Edit: also [submitted](https://news.ycombinator.com/item?id=41678295) to HN in the off chance there's some discussion about experiences.

**Comment by originalnicodr** [comment]
I was wondering where you were! Thanks for the ping, the whole situation sucks :cry: I am so sorry. Will think of ways to avoid situations like this impacting Framed.

Is there anything we can do to help? Will try to see if I can hit up Discord support on Twitter or something.

**Comment by chocmake** [comment]
> Is there anything we can do to help? Will try to see if I can hit up Discord support on Twitter or something.

Hey thanks, that would be really appreciated.

I figured if the HN thread had managed to get the front page some Discord dev may have seen it (I know some post there) but it got auto flagged as self-promotion I think for the first hour or so (likely from having a matching username to the URL).

Not sure if the discordapp subreddit (where some devs also post from time to time) would be similarly strict in that regard, though if someone else felt like posting it there perhaps it may be better received (though being Reddit who knows :p).

Hope everyone at Framed is well :)

**Comment by kapuragu** [comment]
Thank you for your passionate article updating us on what's been going on! I shared it with the server. Incredibly sad state Discord's tech support is in, given how many scam accounts get through checks, while legitimate users have trouble existing.
Someone in the server suggested that r/discordapp had Discord staff visit sometimes, and people have had luck with similar issues by asking them on Twitter. I made a thread with a few @'s here, very much hope that it helps. https://twitter.com/CapLagRobin/status/1840044441027129527
Best of luck, man. Been really missing you

**Comment by notNSANE** [comment]
Oh, thank you for the ping :) Hope the situation gets resolved soon.

I had my issues with Discord due to a false report. Discord team eventually reverted the decision.  In my case, I used Discord's own report tools, sent the appropiate message IDs, evidence, etc. I guess the best chance would be to contact them directly there.

**Comment by chocmake** [comment]
> Thank you for your passionate article updating us on what's been going on! I made a thread with a few @'s here, very much hope that it helps.

Thanks for the kind words and support, cap :heart: 

> I had my issues with Discord due to a false report. Discord team eventually reverted the decision. I guess the best chance would be to contact them directly there.

Yeah, I used the official support form and provided context and details (including some screenshots I happened to have taken of chat prior to it occurring, for reference while creating the 3D file) but there hasn't been any further response via email for the past almost few months now (or anything changed on the account).

**Comment by kapuragu** [comment]
retali8 submitted an additional report weeks ago, which was met by a human-seeming reply telling him to redirect it, and closed the ticket.
![image](https://github.com/user-attachments/assets/de4e6164-2e00-4ace-82c2-32b8c16c1c61)
Then he got another reply, this time this absolutely rage-as-much-as-cringe inducing automated reply.
![image](https://github.com/user-attachments/assets/0eca4001-b76f-4d6d-a70c-38aae87594b4)
And today, doing it again, just 10 minutes afterward, a near-identical reply from Nelly, if that's your real filename, came in.
![image](https://github.com/user-attachments/assets/ce51a1df-d798-427d-bd7e-07f8020497d8)
Absolutely infuriating situation. I shared this on reddit, and not only did the site [double](https://www.reddit.com/r/discordapp/comments/1g8uxsq/for_3_months_now_a_dm_false_positive_restricted/)[post](https://www.reddit.com/r/discordapp/comments/1g8uxm9/for_3_months_now_a_dm_false_positive_restricted/) it, but both immediately were flagged by these mysterious, aetheral "moderators" who read and flag messages faster than I can click one button.
The malevolent AI uprising is here, and we can't do anything to stop it. I'm sorry.

**Comment by chocmake** [comment]
@kapuragu yeah that sucks. Very much appreciate both the efforts in trying. Their support is certainly a black box.

**Comment by kapuragu** [comment]
Happy late new year, man! Since you wrote the article on motions and have done cool stuff with them, figured this might interest you:
https://github.com/TinManTex/mgsv-lookup-strings/pull/13/commits/a88d303004b494b2116324b34fb12a581a2808ae
